tools/calculate_ref_alt_counts.py

```diff
@@ -33,6 +33,56 @@ def get_read_length(bam_cache, sample_cap=20):
 
 
 # Requires BAM files and events list
+def calculate_ref_count(bpp, read_length, reference_genome, bam_cache, buffer=1):
+    """
+    Attempt to calculate the amount of reference reads that support the bpp based on position
+
+    """
+    alt_reads = set()
+    ref_reads = set()
+    ignored_reads = set()
+    multimap_reads = set()
+
+    ref = reference_genome[bpp.break1.chr]
+    try:
+        count, repeat = EventCall.characterize_repeat_region(bpp, reference_genome)
+    except ValueError:
+        # indels cannot be characterized
+        count, repeat = (0, '')
+    # Build the Ref and Alt sequences - This will fail if there are any SNPs in the sample that also occur near the breakpoints.
```

Comment:
Should we be marking a TODO here? for example
```
# Build the Ref and Alt sequences - This will fail if there are any SNPs in the sample that also occur near the breakpoints.
# TODO: Support sequences from complex events
```

Reply:
Sounds good